Sources/MapboxCoreNavigation/NavigationService.swift

```diff
@@ -339,8 +337,6 @@ public class MapboxNavigationService: NSObject, NavigationService {
     
     public func stop() {
         
-        MBXAccounts.resetSession()
```

Comment:
Isn't this required when using MapboxCommon?

Reply:
No, MapboxCommon doesn’t have an analogue to this method and apparently it isn’t needed for the supported billing methods.